src/grids/grid_utils.jl

```diff
@@ -29,20 +29,13 @@ const RingGridOrField = Union{RingGrids.AbstractGrid, RingGrids.AbstractField}
 on_architecture(::GPU, obj::RingGridOrField) = RingGrids.Architectures.on_architecture(RingGrids.Architectures.GPU(), obj)
 on_architecture(::CPU, obj::RingGridOrField) = RingGrids.Architectures.on_architecture(RingGrids.Architectures.CPU(), obj)
 
-# Field utilities
-
-# A bit of type piracy to allow `Field`s to be indexed with tuples
-# TODO: Consider proposing this as a change in Oceananigans
```

Comment:
didi this happen, or did you just remove them like that?

Reply:
Correct